load_sample.py

```diff
@@ -64,8 +86,7 @@ def _create_sample_unit_json(sample_unit_id, sample_unit) -> str:
 
 def main():
```

Comment:
Should we remove this if we're not invoking this from the command line? I think it will fail when we unpack kwargs which don't exist [here](https://github.com/ONSdigital/census-rm-sample-loader/pull/7/files#diff-1e36c58c3e66ea8ab1a9854776f3f3daR32).

Reply:
I don't think it should fail when running locally because they have defaults set in the `RabbitContext()`. Might be an idea to change the rabbit port to the docker dev port instead of the local rabbit it brings up

Reply:
Ah good point, that will explain why it failed for me then!